README.md

```diff
@@ -118,71 +135,79 @@ vault write data/buildkite/my_pipeline/git-credentials value=- <<< $(echo "https
 These are then exposed via a [gitcredential helper](https://git-scm.com/docs/gitcredentials) which will download the
 credentials as needed.
 
-### Environment variables
+## Options
+***
+The Vault Secrets plugin supports a number of different configuration options.
 
-Key values pairs can also be uploaded.
+### `server`
+The address of the target Vault server. Example: `https://my-vault-server:8200`
 
-```bash
-vault write data/buildkite/my_pipeline/environment value=- <<< $(echo "MY_SECRET=blah" | base64)
-```
+### `path`
+Alternative Base Path to use for Vault secrets. This is expected to be a [KV Store](https://developer.hashicorp.com/vault/docs/secrets/kv#kv-version-2)  
 
-```bash
-vault write data/buildkite/my_pipeline/env_key value=- <<< $(echo "my secret"| base64)
-```
+Defaults to: `data/buildkite`
 
-Can be loaded using:
+
+### `auth`
+Configure the authentication parameters the plugin should use to authenticate with Vault.
```

Comment:
I would clarify that this is expected to be a dictionary/map and remove one of the two times `authenticate` is used for easier reading:
```suggestion
Dictionary/map with the configuration of the parameters the plugin should use to authenticate with Vault.
```